### PR TITLE
Add meeting details to chat

### PR DESCRIPTION
**Pull request by jessiechan27**

### CODE REVIEW

File: api-reference/beta/api/chat-get.md [hunk 1]
```diff
@@ -87,7 +87,14 @@ Content-type: application/json
     "topic": "test group 1",
     "createdDateTime": "2021-04-06T19:49:52.431Z",
     "lastUpdatedDateTime": "2021-04-06T19:54:04.306Z",
-    "chatType": "group"
+    "chatType": "group",
+    "webUrl": "https://teams.microsoft.com/l/chat/19%3Ad74fc2ed-cb0e-4288-a219-b5c71abaf2aa_8c0a1a67-50ce-4114-bb6c-da9c5dbcf6ca@unq.gbl.spaces/0?tenantId=b33cbe9f-8ebe-4f2a-912b-7e2a427f477f"
```
[the remark this answers is not in the document]
Comment (jessiechan27) [reply api-reference/beta/api/chat-get.md:91]: the webUrl does contain chat id within the URL, but the webUrl also contains the teams URL, tenant id and an index to a message in the chat. They're both necessary

File: api-reference/beta/api/chat-get.md [hunk 2]
```diff
@@ -264,6 +283,54 @@ Content-type: application/json
 }
 ```
 
+### Example 4: Get a meeting chat
```
Comment (AkJo) [inline api-reference/beta/api/chat-get.md:286]: Get the meeting details of a chat.

File: api-reference/beta/resources/chat.md [hunk 3]
```diff
@@ -64,8 +64,10 @@ not all scenarios are possible. It is possible to get chat IDs with delegated pe
 | createdDateTime| dateTimeOffset|  Date and time at which the chat was created. Read-only.|
 | lastUpdatedDateTime| dateTimeOffset|  Date and time at which the chat was renamed or list of members were last changed. Read-only.|
 | chatType| [chatType](../resources/chat.md#chattype-values) | Specifies the type of chat. Possible values are:`group`, `oneOnOne` and `meeting`.|
-| viewpoint|[chatViewpoint](../resources/chatviewpoint.md)|Represents caller-specific information about the chat, such as last message read date and time. This property is populated only when the request is made in a delegated context.|
 | webUrl| String | A hyperlink that will go to the chat in Microsoft Teams. This URL should be treated as an opaque blob, and not parsed. Read-only.|
+| tenantId| String | The tenant identifier. Read-only.|
```
Comment (AkJo) [inline api-reference/beta/resources/chat.md:68]: The identifier of the tenant in which the chat was created.

File: api-reference/beta/resources/teamworkonlinemeetinginfo.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+``` json
+{
+  "calendarEventId": "string",
+  "joinWebUrl": "string",
+  "organizer": {"@odata.type": "microsoft.graph.teamworkUserIdentity"}
+}
+```
+
+## See also
+- [chat](chat.md)
```
Comment (AkJo) [inline api-reference/beta/resources/teamworkonlinemeetinginfo.md:44]: Isnt the casing wrong?

File: api-reference/beta/resources/chat.md [hunk 5]
```diff
@@ -64,8 +64,10 @@ not all scenarios are possible. It is possible to get chat IDs with delegated pe
...
 | lastUpdatedDateTime| dateTimeOffset|  Date and time at which the chat was renamed or list of members were last changed. Read-only.|
 | chatType| [chatType](../resources/chat.md#chattype-values) | Specifies the type of chat. Possible values are:`group`, `oneOnOne` and `meeting`.|
-| viewpoint|[chatViewpoint](../resources/chatviewpoint.md)|Represents caller-specific information about the chat, such as last message read date and time. This property is populated only when the request is made in a delegated context.|
 | webUrl| String | A hyperlink that will go to the chat in Microsoft Teams. This URL should be treated as an opaque blob, and not parsed. Read-only.|
+| tenantId| String | The identifier of the tenant in which the chat was created. Read-only.|
+| viewpoint|[chatViewpoint](../resources/chatviewpoint.md)|Represents caller-specific information about the chat, such as last message read date and time. This property is populated only when the request is made in a delegated context.|
+| onlineMeetingInfo | [teamworkOnlineMeetingInfo](../resources/teamworkonlinemeetinginfo.md) | Details of the online meeting if the chat is associated with an online meeting. Read-only.|
```
Comment (AkJo) [inline api-reference/beta/resources/chat.md:70]: @Lauragra for wordsmithing.

**Review by AkJo — APPROVED** [review]

File: api-reference/beta/resources/chat.md (hunk 5 above)
Comment (JarbasHorst) [inline api-reference/beta/resources/chat.md:70]: I'm not Laura but here my suggestion:

- Represents details about an online meeting. If the chat isn't associated with an online meeting, the property is empty.

File: api-reference/beta/resources/teamworkhostedcontent.md [hunk 6]
```diff
@@ -42,7 +42,6 @@ The following is a JSON representation of the resource.
 -->
 ``` json
 {
-  "@odata.type": "#microsoft.graph.teamworkHostedContent",
```
Comment (JarbasHorst) [inline api-reference/beta/resources/teamworkhostedcontent.md:45]: Why do you want to remove the @odata.type property from the JSON representation?

File: api-reference/beta/resources/teamworkonlinemeetinginfo.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
+---
+title: "teamworkOnlineMeetingInfo resource type"
+description: "Represents details about an online Teams meeting"
```
Comment (JarbasHorst) [inline api-reference/beta/resources/teamworkonlinemeetinginfo.md:3]: For clarity, please add Teams after meeting and replace Teams with Microsoft Teams. Also, add a period at the end of the sentence. Suggested revision:

- Represents details about an online meeting in Microsoft Teams.

File: api-reference/beta/resources/teamworkonlinemeetinginfo.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+doc_type: resourcePageType
+---
+
+# teamworkOnlineMeetingInfo resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents details about an online Teams meeting.
```
Comment (JarbasHorst) [inline api-reference/beta/resources/teamworkonlinemeetinginfo.md:16]: For clarity, please add Teams after meeting and replace Teams with Microsoft Teams. Suggested revision:

- Represents details about an online meeting in Microsoft Teams.

File: changelog/Microsoft.Teams.GraphSvc.json [hunk 9]
```diff
@@ -135,34 +135,72 @@
...
       "Version": "beta",
       "CreatedDateTime": "2021-06-07T11:01:30.771Z",
       "WorkloadArea": "Teamwork",
       "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "ef72d160-05bc-4f45-9ae9-2bdb84dd8c07",
+          "ApiChange": "ComplexType",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Teams.GraphSvc.json:175]: ComplexType is not on the list of approved values for ApiChange. Please change this to "Resource".

File: changelog/Microsoft.Teams.GraphSvc.json [hunk 10]
```diff
@@ -135,34 +135,72 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the [teamworkOnlineMeetingInfo](https://docs.microsoft.com/en-us/graph/api/resources/teamworkOnlineMeetingInfo?view=graph-rest-beta) resource type.",
+          "Target": "teamworkOnlineMeetingInfo"
+        },
+        {
+          "Id": "ef72d160-05bc-4f45-9ae9-2bdb84dd8c07",
+          "ApiChange": "Property",
+          "ChangedApiName": "onlineMeetingInfo",
+          "ChangeType": "Addition",
+          "Description": "Added the **onlineMeetingInfo** property to [chat](https://docs.microsoft.com/en-us/graph/api/resources/chat?view=graph-rest-beta) resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Teams.GraphSvc.json:186]: Articles help readers and translation software identify the nouns and modifiers in a sentence. Please, add the before chat. Suggested revision:

- ...property to the [chat]...

File: changelog/Microsoft.Teams.GraphSvc.json [hunk 11]
```diff
@@ -135,34 +135,72 @@
...
+          "Target": "chat"
+        },
+        {
+          "Id": "ef72d160-05bc-4f45-9ae9-2bdb84dd8c07",
+          "ApiChange": "Property",
+          "ChangedApiName": "tenantId",
+          "ChangeType": "Addition",
+          "Description": "Added the **tenantId** property to [chat](https://docs.microsoft.com/en-us/graph/api/resources/chat?view=graph-rest-beta) resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Teams.GraphSvc.json:194]: Articles help readers and translation software identify the nouns and modifiers in a sentence. Please, add the before chat. Suggested revision:

- ...property to the [chat]...

File: changelog/Microsoft.Teams.GraphSvc.json [hunk 12]
```diff
@@ -135,34 +135,72 @@
...
+          "Target": "chat"
+        }
+      ],
+      "Id": "1c5850b2-105c-4d50-a378-ac9bb52a36c7",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Teams.GraphSvc.json:198]: Within a changelog record, all Id properties must have the same value. Please, replace 1c5850b2-105c-4d50-a378-ac9bb52a36c7 with ef72d160-05bc-4f45-9ae9-2bdb84dd8c07

File: api-reference/beta/api/chat-get.md [hunk 13]
```diff
@@ -264,6 +283,54 @@ Content-type: application/json
 }
 ```
 
+### Example 4: Get the meeting details of a chat associated with a Teams meeting
```
Comment (JarbasHorst) [inline api-reference/beta/api/chat-get.md:286]: For clarity, I recommend you to add Teams after meeting and replace Teams with Microsoft Teams. Suggested revision:

- ...with a meeting in Microsoft Teams

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: api-reference/beta/resources/teamworkhostedcontent.md (hunk 6 above)
Comment (jessiechan27) [reply api-reference/beta/resources/teamworkhostedcontent.md:45]: it wasn't supposed to be there originally. I was looking at similar files and trying to make them consistent

**Review by Lauragra — APPROVED** [review]

**Pull request merged**